### PR TITLE
add Fistful of flowers items, PCs and NPCs

### PR DESCRIPTION
**Pull request by TMunsie**

### CODE REVIEW

**Comment by TikaelSol** [comment]
I can never remember if we get to add these before the PDF is actually out.

**Comment by TMunsie** [comment]
> I can never remember if we get to add these before the PDF is actually out.



> I can never remember if we get to add these before the PDF is actually out.

It is being run at conventions, which is our "new bar". I'll be adding a bunch of PFS stuff pre-GenCon for the same reason :) The Free RPG dayu adventure is allowed to be run online/using Foundry IF the players and GM could have run it at the same FLGS. Since Foundry can be run offline this fits within our new, friendly-with-Paizo rules since no lore/adventure content is going in.

File: packs/data/one-shot-bestiary.db/meliosas-leshy-golden-bamboo.json [hunk 1]
```diff
@@ -0,0 +1,668 @@
...
+        },
+        "traits": {
+            "di": {
+                "custom": "",
+                "value": []
+            },
+            "dr": [],
+            "dv": [],
+            "languages": {
+                "custom": "speak with plants (same type as leshy)",
```
Comment (SpartanCPA) [inline packs/data/one-shot-bestiary.db/meliosas-leshy-golden-bamboo.json:87]:
```suggestion
                "custom": "Speak with Plants (Golden Bamboo Only)",
```

File: packs/data/one-shot-bestiary.db/meliosas-leshy-golden-bamboo.json [hunk 2]
```diff
@@ -0,0 +1,668 @@
...
+            "data": {
+                "actionCategory": {
+                    "value": "defensive"
+                },
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
```
Comment (SpartanCPA) [inline packs/data/one-shot-bestiary.db/meliosas-leshy-golden-bamboo.json:430]:
```suggestion
                },
                "deathNote": true,
```

File: packs/data/one-shot-bestiary.db/meliosas-leshy.json [hunk 3]
```diff
@@ -0,0 +1,594 @@
...
+                    "value": "defensive"
+                },
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "description": {
+                    "value": "<p>When a leshy dies, a burst of primal energy explodes from its body, restoring [[/r 2d8 # hit points]]{2d8 Hit Points} to each plant creature within a @Template[type:emanation|distance:30]. This area is filled with plants of the same type as the leshy, becoming difficult terrain. If the terrain isn't a viable environment for these plants, they wither after 24 hours.</p>"
```
Comment (SpartanCPA) [inline packs/data/one-shot-bestiary.db/meliosas-leshy.json:432]:
```suggestion
                    "value": "<p>When a leshy dies, a burst of primal energy explodes from its body, restoring [[/r {2d8}[healing]]]{2d8 Hit Points} to each plant creature within a @Template[type:emanation|distance:30]. This area is filled with plants of the same type as the leshy, becoming difficult terrain. If the terrain isn't a viable environment for these plants, they wither after 24 hours.</p>"
```

File: packs/data/one-shot-bestiary.db/meliosas-leshy.json [hunk 4]
```diff
@@ -0,0 +1,594 @@
...
+            "data": {
+                "actionCategory": {
+                    "value": "defensive"
+                },
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
```
Comment (SpartanCPA) [inline packs/data/one-shot-bestiary.db/meliosas-leshy.json:430]:
```suggestion
                },
                "deathNote": true,
```

File: packs/data/one-shot-bestiary.db/meliosas-leshy-minkaian-honeysuckle-leshy.json [hunk 5]
```diff
@@ -0,0 +1,668 @@
...
+            "dr": [
+                {
+                    "exceptions": "",
+                    "type": "physical",
+                    "value": 3
+                }
+            ],
+            "dv": [],
+            "languages": {
+                "custom": "speak with plants (same type as leshy)",
```
Comment (SpartanCPA) [inline packs/data/one-shot-bestiary.db/meliosas-leshy-minkaian-honeysuckle-leshy.json:93]:
```suggestion
                "custom": "Speak with Plants (Minkaian Honeysuckle Only)",
```

File: packs/data/one-shot-bestiary.db/meliosas-leshy-minkaian-honeysuckle-leshy.json [hunk 6]
```diff
@@ -0,0 +1,668 @@
...
+            "data": {
+                "actionCategory": {
+                    "value": "defensive"
+                },
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
```
Comment (SpartanCPA) [inline packs/data/one-shot-bestiary.db/meliosas-leshy-minkaian-honeysuckle-leshy.json:504]:
```suggestion
                },
                "deathNote": true,
```

File: packs/data/one-shot-bestiary.db/meliosas-leshy-wisteria-leshy.json [hunk 7]
```diff
@@ -0,0 +1,687 @@
...
+            "data": {
+                "actionCategory": {
+                    "value": "defensive"
+                },
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
```
Comment (SpartanCPA) [inline packs/data/one-shot-bestiary.db/meliosas-leshy-wisteria-leshy.json:433]:
```suggestion
                },
                "deathNote": true,
```

File: packs/data/one-shot-bestiary.db/meliosas-leshy-wisteria-leshy.json [hunk 8]
```diff
@@ -0,0 +1,687 @@
...
+                "description": {
+                    "value": "<p>When a leshy dies, a burst of primal energy explodes from its body, restoring [[/r 2d8 # hit points]]{2d8 Hit Points} to each plant creature within a @Template[type:emanation|distance:30]. This area is filled with plants of the same type as the leshy, becoming difficult terrain. If the terrain isn't a viable environment for these plants, they wither after 24 hours.</p>"
```
Comment (SpartanCPA) [inline packs/data/one-shot-bestiary.db/meliosas-leshy-wisteria-leshy.json:435]:
```suggestion
                    "value": "<p>When a leshy dies, a burst of primal energy explodes from its body, restoring [[/r {2d8}[healing]]]{2d8 Hit Points} to each plant creature within a @Template[type:emanation|distance:30]. This area is filled with plants of the same type as the leshy, becoming difficult terrain. If the terrain isn't a viable environment for these plants, they wither after 24 hours.</p>"
```

File: packs/data/one-shot-bestiary.db/scented-candle-homunculus.json [hunk 9]
```diff
@@ -0,0 +1,451 @@
...
+                    "value": "defensive"
+                },
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "description": {
+                    "value": "<p>@Compendium[pf2e.bestiary-ability-glossary-srd.Aura]{Aura} 5 feet.</p>\n<p>The scented candle homunculus's fragrance of lavender is overwhelmingly intense. Any creature other than the homunculus's master who enters the aura must attempt a @Check[type:fortitude|dc:15] save.</p>\n<hr />\n<p><strong>Critical Success</strong> The creature is temporarily immune to the homunculus's aura for 24 hours.</p>\n<p><strong>Success</strong> The creature is unaffected.</p>\n<p><strong>Failure</strong> The creature is @Compendium[pf2e.conditionitems.Sickened]{Sickened 1}.</p>\n<p><strong>Critical Failure</strong> The creature is @Compendium[pf2e.conditionitems.Stupefied]{Stupefied 1} for 1 round and sickened 1.</p>"
```
Comment (SpartanCPA) [inline packs/data/one-shot-bestiary.db/scented-candle-homunculus.json:318]:
```suggestion
                    "value": "<p>@Template[type:emanation|distance:5]{5 feet} @Compendium[pf2e.bestiary-ability-glossary-srd.Aura]{Aura}</p>\n<p>The scented candle homunculus's fragrance of lavender is overwhelmingly intense. Any creature other than the homunculus's master who enters the aura must attempt a @Check[type:fortitude|dc:15] save.</p>\n<hr />\n<p><strong>Critical Success</strong> The creature is temporarily immune to the homunculus's aura for 24 hours.</p>\n<p><strong>Success</strong> The creature is unaffected.</p>\n<p><strong>Failure</strong> The creature is @Compendium[pf2e.conditionitems.Sickened]{Sickened 1}.</p>\n<p><strong>Critical Failure</strong> The creature is @Compendium[pf2e.conditionitems.Stupefied]{Stupefied 1} for 1 round and sickened 1.</p>"
```

File: packs/data/one-shot-bestiary.db/scented-candle-homunculus.json [hunk 10]
```diff
@@ -0,0 +1,451 @@
...
+                    "value": "offensive"
+                },
+                "actionType": {
+                    "value": "action"
+                },
+                "actions": {
+                    "value": 2
+                },
+                "description": {
+                    "value": "<p>A scented candle homunculus who is adjacent to the large wax cauldron can dunk itself in wax, [[/r {2d6} # hit points]]{healing 2d6 damage}.</p>"
```
Comment (SpartanCPA) [inline packs/data/one-shot-bestiary.db/scented-candle-homunculus.json:359]:
```suggestion
                    "value": "<p>A scented candle homunculus who is adjacent to the large wax cauldron can dunk itself in wax, [[/r {2d6}[healing]]]{healing 2d6 damage}.</p>"
```

**Review by SpartanCPA — CHANGES_REQUESTED** [review]

**Review by SpartanCPA — APPROVED** [review]

**Pull request merged**